GliaWidgets/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -852,6 +920,7 @@ extension ChatViewModel {
         case choiceOptionSelected(ChatChoiceCardOption, String)
         case chatScrolled(bottomReached: Bool)
         case linkTapped(URL)
+        case customCardOptionSelected(HTMLMetadata.Option, String)
```

Comment:
I would suggest to add named arguments for clarity: `.customCardOptionSelected(HTMLMetadata.Option,  messageId: Message.Id)`

Reply:
475b1f015cab5490a7d22fec77c4baa1ec006549